zilencer/management/commands/populate_db.py

```diff
@@ -283,7 +283,8 @@ def handle(self, **options: Any) -> None:
             iago.save(update_fields=['is_staff'])
 
             desdemona = get_user_by_delivery_email("desdemona@zulip.com", zulip_realm)
-            do_change_user_role(desdemona, UserProfile.ROLE_REALM_ADMINISTRATOR)
+
+            do_change_user_role(desdemona, UserProfile.ROLE_REALM_OWNER)
```

Comment:
Remove the extra space.  But I agree with the decision to tag Iago as a realm administrator (but not owner); that limits how much we need to change tests.